tests/x11/rrdtool_x11.pm

```diff
@@ -0,0 +1,89 @@
+# SUSE"s openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Regression test for rrdtool
+# - created, update rrd file
+# - fetch data from file
+# - generate images with different options
+# - compare images with needles
+# Expected result:
+# 4 graphics will be displayed without error.
+#
+# Maintainer: Marcelo Martins <mmartins@suse.cz>
+
+use base "x11test";
+use strict;
+use warnings;
+use testapi;
+use utils;
+use version_utils 'is_sle';
+
+sub verify_rrd_image {
+    my $rrdtool_image = shift;
+    send_key("alt-f2");
+    type_string("eog /tmp/rrdtool/${rrdtool_image}.png\n");
+    assert_screen "rrdtool_image_${rrdtool_image}";
+    send_key 'alt-f4';
+    assert_screen "rrdtool_xterm";
+}
+
+sub rrdtool_update {
+    my $data = shift;
+    assert_script_run "rrdtool update test.rrd ${data}";
+}
+
+sub run {
+    select_console 'x11';
+    x11_start_program('xterm');
+    become_root;
+    pkcon_quit;
+    # create a tmp dir/files to work
+    assert_script_run "mkdir /tmp/rrdtool; cd /tmp/rrdtool";
+    # install requirements
+    zypper_call "in rrdtool";
+    # create a rrd file
+    assert_script_run "rrdtool create test.rrd --start 920804400  DS:speed:COUNTER:600:U:U RRA:AVERAGE:0.5:1:24 RRA:AVERAGE:0.5:6:10";
```

Comment:
I would have used a script to generate all of this, and just call `verify_rrd_image` later, so that there's less chance of having tests failing due to missing keys or things like that. But Looks good altogether :+1:

Reply:
I change one line to remove timeout, as @czerw requested. I dont know if I delayed to this